protos/fpc/fpc.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. All Rights Reserved.
+// Copyright 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package fpc;
+option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos";
+
+message CC_Parameters {
+    string chaincode_id   = 1;      // chaincode name
+    string version        = 2;      // chaincode version, this is the expected mrenclave
+    string sequence       = 3;      // chaincode sequence number
+    string channel_id     = 4;      // channel id
+}
+
+message Host_Parameters {
+    bytes peer_identity = 1;        // peer that hosts the enclave
+}
+
+message Attested_Data {
+    CC_Parameters cc_params    = 1;
+    Host_Parameters host_params = 2;
+    bytes enclave_vk           = 3; // chaincode enclave public key
+    bytes channel_hash         = 4; // cryptographic hash of the channel genesis block;
+    string tlcc_mrenclave      = 5; // expected TLCC mrenclave
+}
+
+message Credentials {
+    google.protobuf.Any serialized_attested_data   = 1;  // serialization of type **Attested_Data**
+    bytes attestation           = 2;    // optional attestation/quote
```

Comment:
purely a style suggestion but we either call both of them `..._bytes` or drop it from both?

Reply:
dropped